src/sentry/api/serializers/models/group.py

```diff
@@ -226,42 +235,58 @@ def get_attrs(self, item_list, user):
 
         seen_stats = self._get_seen_stats(item_list, user)
 
+        annotations_by_group_id = defaultdict(list)
+
+        # find every org that's part of the item list
+        organization_id_list = list(set(item.project.organization_id for item in item_list))
```

Comment:
Do you need to loop the items? Shouldn't all the groups come from a single org?

Reply:
@markstory I can't imagine that there would be more than one organization. That being said, I don't see the harm in doing this just to be safe. All it does is change the subsequent query from `organiztion=` to `organizations__in=`. I still have to loop through the installs.

Reply:
@MeredithAnya Ok. I will just use the first org. But I will log an error if there is more than one so if by chance this ever happens, we'll know about it.